src/src-admin/views/ExcelUploader/index.js

```diff
@@ -0,0 +1,303 @@
+import React, { useState } from 'react';
```

Comment:
Lots of commented out code throughout this file that I'm not sure has any intention of being used. Can go ahead and remove and should help tidy it up a good bit